src/services/locale.ts

```diff
@@ -0,0 +1,127 @@
+     * suggested currency. You can use the locale header to get the data in a
+     * supported language.
+     * 
+     * ([IP Geolocation by DB-IP](https://db-ip.com))
+     *
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async get(): Promise<string> {
+        let path = '/locale';
+        
+        return await this.client.call('get', path, {
+                    'content-type': 'application/json',
+               },
+               {
+            });
+    }
+
+    /**
+     * List Countries
+     *
+     * List of all continents. You can use the locale header to get the data in a
+     * supported language.
+     *
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async getContinents(): Promise<string> {
+        let path = '/locale/continents';
+        
+        return await this.client.call('get', path, {
+                    'content-type': 'application/json',
+               },
+               {
+            });
+    }
+
+    /**
+     * List Countries
+     *
+     * List of all countries. You can use the locale header to get the data in a
+     * supported language.
+     *
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async getCountries(): Promise<string> {
+        let path = '/locale/countries';
+        
+        return await this.client.call('get', path, {
+                    'content-type': 'application/json',
+               },
+               {
+            });
+    }
+
+    /**
+     * List EU Countries
+     *
+     * List of all countries that are currently members of the EU. You can use the
+     * locale header to get the data in a supported language.
+     *
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async getCountriesEU(): Promise<string> {
+        let path = '/locale/countries/eu';
+        
+        return await this.client.call('get', path, {
+                    'content-type': 'application/json',
+               },
+               {
+            });
+    }
+
+    /**
+     * List Countries Phone Codes
+     *
+     * List of all countries phone codes. You can use the locale header to get the
+     * data in a supported language.
+     *
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async getCountriesPhones(): Promise<string> {
+        let path = '/locale/countries/phones';
+        
+        return await this.client.call('get', path, {
+                    'content-type': 'application/json',
+               },
+               {
+            });
+    }
+
+    /**
+     * List Currencies
+     *
+     * List of all currencies, including currency symol, name, plural, and decimal
+     * digits for all major and minor currencies. You can use the locale header to
+     * get the data in a supported language.
+     *
+     * @throws Exception
+     * @return Promise<string>
+     */
+    async getCurrencies(): Promise<string> {
```

Comment:
Another general design suggestion, you can create the entiries for each of your services and use `get` with the field values:

```ts
   enum LocaleFields = {
      currencies = 'currencies'
      countriesPhones = 'phones'
   }

get(field: LocaleFields, context?: any) {
  let path = '/locale/${field}';

.... you can also pass context data  for selected fields if necessary
}
```

You can then create aliases if you'd like, but anyhow - it'll save you tons of repetitive code.